core/src/avm2/globals/flash/system/JPEGLoaderContext.as

```diff
@@ -5,6 +5,7 @@
 
 package flash.system
 {
+    [API("663")]
```

Comment:
It's possible that the failing test has an SWF version which shouldn't have this class. Can you check whether it works in Flash Player?

Reply:
It does work in plain Flash Player 32. So, wat:
![image](https://github.com/user-attachments/assets/b3d7f757-c084-40cc-ac84-fa9f74038604)

I suggest just leaving this as a note in a comment...

Reply:
What about just `[API("662")]`? Then, like FP, it'll only be accessible in SWFv10 and above.

Reply:
sure, I will change to [API("662")], but I am not sure what to write as a note.
```
In playerglobal.swc it is [API("663")], but because of the failed jpeg_loader_context test changed to [API("662")]
```
?

Reply:
I don't think the change was because of the test - it was because it's actually available in FP, the test just (accidentally, but thankfully) pointed that fact out.
Maybe something like this?
`// Both the docs and playerglobal.swc says this is "663", but it's actually available in regular Flash Player...`

Reply:
sounds good, thanks